docs/api/ref/parameters/product_available-fields.yaml

```diff
@@ -0,0 +1,224 @@
+components:
+  parameters:
+    ProductAvailableFields:
+      name: fields
+      in: query
+      description: Specific fields to return. Use 'knowledge_panels' for Knowledge Panels only.
+      required: false
+      style: form
+      # Style 'form' with explode=false transforms the array into a comma-separated query string
+      explode: false
+      schema:
+        type: array
+        items:
+          type: string
+          enum:
+            # meta/special fields
+            - knowledge_panels
+            # all other fields
+            - added_countries_tags
+            - additives_n
+            - additives_original_tags
+            - additives_tags
+            - allergens
+            - allergens_from_ingredients
+            - allergens_from_user
+            - allergens_hierarchy
+            - allergens_tags
+            - amino_acids_prev_tags
+            - amino_acids_tags
+            - brands
+            - brands_tags
+            - categories
+            - categories_hierarchy
+            - categories_lc
+            - categories_old
+            - categories_properties
+            - categories_properties_tags
+            - categories_tags
+            - category_properties
+            - checkers_tags
+            - ciqual_food_name_tags
+            - cities_tags
+            - code
+            - codes_tags
+            - compared_to_category
+            - complete
+            - completeness
+            - correctors_tags
+            - countries
+            - countries_hierarchy
+            - countries_lc
+            - countries_tags
+            - created_t
+            - creator
+            - data_quality_bugs_tags
+            - data_quality_errors_tags
+            - data_quality_info_tags
+            - data_quality_tags
+            - data_quality_warnings_tags
+            - ecoscore_data
+            - ecoscore_extended_data
+            - ecoscore_extended_data_version
+            - ecoscore_grade
+            - ecoscore_score
+            - ecoscore_tags
+            - editors_tags
+            - emb_codes
+            - emb_codes_tags
+            - entry_dates_tags
+            - expiration_date
+            - food_groups
+            - food_groups_tags
+            - generic_name
+            - generic_name_it
```

Comment:
all the fiields like generic_name_it, product_name_it, ingredients_text_it etc. are for Italian, but they can be requested in any language, e.g. product_name_en for English
So having a comprehensive enum is difficult (unless we list all the hundreds of different possibilities, but I'm not sure that's a good idea)

Reply:
You are right. I replaced the definition with enum with one with string pattern. try to see if it could work.
Thanks a lot
Luigi